caikit/core/data_model/dataobject.py

```diff
@@ -270,31 +292,32 @@ def _get_all_enums(
     return all_enums
 
 
-def _make_data_model_class(proto_class, wrapped_cls):
-    wrapper_cls = _DataBaseMetaClass(
-        wrapped_cls.__name__,
-        tuple([DataBase, wrapped_cls]),
-        {"_proto_class": proto_class, **wrapped_cls.__dict__},
-    )
-    update_wrapper(wrapper_cls, wrapped_cls, updated=())
+def _make_data_model_class(proto_class, cls):
+    if issubclass(cls, DataObjectBase):
+        _DataBaseMetaClass.parse_proto_descriptor(cls)
```

Comment:
This is where the lazy proto parsing happens now that the protobuf class has been created with `dataclass_to_proto`